requirements-core.txt

```diff
@@ -0,0 +1,21 @@
+attrs>=19.3.0
+defusedxml>=0.6.0
+GitPython>=3.0.8
+lxml>=4.4.1
+matplotlib>=3.3.1
+numpy>=1.17.3
+Pillow>=6.1.0
+
+# Avoid 2.0.2 Linux binary distribution because of
```

Comment:
Technically, in requirements.txt we could just use --no-binary, and I'd prefer this solution. But it is not supported in setup.py.